collector/config/configgrpc/Makefile

```diff
@@ -0,0 +1 @@
+include ../../Makefile.Common
```

Comment:
note: this belongs to the upstream repo, and has no effect in odigos repo. It is here only to make future upgrades easier, so there are minimum changes between us and upstream